about.css

```diff
@@ -0,0 +1,465 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:ital,wght@1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-family: 'lato', sans-serif;
+  list-style: none;
+  text-decoration: none;
+}
+
+body {
+  width: 100vw;
+}
+
+/* Header and Section 1 */
+
+.header-section1-container {
+  background-image: url('images/home-bg-mobile.png');
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  padding: 20px 5vw;
+}
+
+/* Header */
+
+.header .hamburger {
+  display: inline-block;
+  cursor: pointer;
+}
+
+.header .hamburger span {
+  display: block;
+  height: 5px;
+  width: 25px;
+  margin-bottom: 5px;
+  background: #333;
+}
+
+.header .mobile-nav {
+  z-index: 100;
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: none;
+  width: 100vw;
+  height: 100vh;
+  justify-content: space-between;
+  background: #e4e4e4;
+  font-weight: 900;
+  font-size: 40px;
+  line-height: 60px;
+}
+
+.header .mobile-nav .a {
+  position: relative;
+  left: 20px;
+  top: 50px;
+  color: rgb(111, 110, 110);
+}
+
+.header .mobile-nav .closeBtn {
+  color: grey;
+  position: relative;
+  right: 20px;
+  cursor: pointer;
+}
+
+.header .mobile-nav .a:hover,
+.header .mobile-nav .closeBtn:hover {
+  color: #ec5242;
+}
+
+.header .mobile-nav .li-cc {
+  margin-top: 30px;
+}
+
+.header .mobile-nav .a-cc {
+  border: solid 4px red;
+  padding: 10px;
+  color: #ec5242;
+  white-space: nowrap;
+}
+
+.header .desktop-nav {
+  display: none;
+}
+
+/* Section 1 */
+
+.section-1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+  margin: 60px 0;
+  text-align: center;
+}
+
+.section-1 .hello {
+  color: #ec5242;
+  font-size: larger;
+}
+
+.section-1 .h1 {
+  color: #ec5242;
+  font-size: 40px;
+  line-height: 40px;
+  margin-bottom: 20px;
+}
+
+.section-1 .p {
+  background: white;
+  border: 1px solid #d3d3d3;
+  padding: 20px;
+  line-height: 25px;
+}
+
+.section-1 .desktop-p {
+  display: none;
+}
+
+.section-1 .email {
+  color: #333;
+  text-decoration: underline;
+  text-underline-position: under;
+  padding-bottom: 5px;
+  font-size: larger;
+  font-weight: 500;
+  margin-top: -10px;
+}
+
+/* Section 2 */
+
+.section-2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 50px;
+  padding: 50px 5vw;
+  text-align: center;
+}
+
+.section-2 .text-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 15px;
+  padding: 0 30px;
+}
+
+.section-2 .red-line {
+  height: 1px;
+  width: 25px;
+  background: #ec5242;
+}
+
+.section-2 .desktop-p {
+  display: none;
+}
```

Comment:
- [ ] Kinldy make sure this paragraph element is displayed on the mobile version as shown in the original design as it's creating a discrepancy between your design and the original one.
- Your design: 
  ![image](https://github.com/suthish777/capstone__01/assets/104968236/b966204e-0e51-42ba-b96b-999d3649988b)
- Original design:
  ![image](https://github.com/suthish777/capstone__01/assets/104968236/34edb369-ad56-4c25-9c8f-5016d8020921)